docs/android/device/setup.md

```diff
@@ -0,0 +1,114 @@
+
+It's possible to debug an android device over WiFi, without keeping the device physically connected to the computer. This technique requires more effort, but could be useful when the device is too far from the computer to remain constantly plugged-in via a cable.
+
+### Connecting over WiFi
+
+By default, the [Android Debug Bridge](https://developer.android.com/tools/help/adb.html) (adb) is configured to communicate with an Android device via USB. It's possible to reconfigure it to use TCP/IP instead of USB. To do this, both the device and the computer must be on the same WiFi network.
+
+First, enable Wireless debugging on your Android device:
+
+01. Follow the steps in the [Enable developer mode on the device](#enable-developer-mode-on-the-device) section.
+01. Follow the steps in the [Enable USB debugging](#enable-usb-debugging) section.
+01. Go to the **Settings** screen.
+01. Select **Developer options**.
+01. Turn on the **Wireless debugging** option.
+
+Depending on the UI your device is running, the **Wireless debugging** option may be in a different location. Consult your device documentation if you can't find **Wireless debugging**.
+
+Next, use adb to connect to your device, first through a USB connection:
+
+01. Determine the IP address of your Android device. One way to find out the IP address is to look under **Settings > Network & internet > Wi-Fi**, then tap on the WiFi network that the device is connected to, and then tap on **Advanced**. This will open a dropdown showing information about the network connection, similar to what is seen in the screenshot below:
+
+    :::image type="content" source="media/setup/ip-settings-sml.png" alt-text="Android status screen with IP address." lightbox="media/setup/ip-settings.png":::
+
+    On some versions of Android the IP address won't be listed there but can be found instead under **Settings > About phone > Status**.
+
+01. In Visual Studio, open the adb command prompt by selecting the menu option: **Tools** > **Android** > **Android Adb Command Prompt...**.
+
+01. In the command prompt, use the `adb tcpip` command to tell the device to listen to TCP/IP connections on port 5555.
+
+    ```command
+    adb tcpip 5555
+    ```
+
+01. Disconnect the USB cable from your device.
+
+01. Connect to the device's IP address with port 5555:
+
+    ```command
+    adb connect 192.168.1.28:5555
+    ```
+
+    When this command finishes, the Android device is connected to the computer via WiFi.
+
+    When you're finished debugging via WiFi, you can reset ADB back to USB mode with the following command:
+
+    ```command
+    adb usb
+    ```
+
+    To see the devices connected to the computer, use the `adb devices` command:
+
+    ```command
+    adb devices
+    ```
+
+## Summary
+
+This article discussed how to configure an Android device for development by enabling debugging on the device. It also covered how to connect the device to a computer using either USB or WiFi.
+
+## Related Links
```

Comment:
Always delete the related links section.

Reply:
If these links are worth keeping, do something like the debug-on-emulator doc that ends with "For more info about....see the following..."?